internal/quickstart/messages.go

```diff
@@ -59,7 +61,14 @@ func sendCreateFlagMsg(client flags.Client, accessToken, baseUri, flagName, flag
 			projKey,
 		)
 		if err != nil {
-			return errMsg{err: err}
+			var e struct {
+				Code    string `json:"code"`
+				Message string `json:"message"`
+			}
+			_ = json.Unmarshal([]byte(err.Error()), &e)
+			if e.Code != "conflict" {
+				return errMsg{err: errors.NewError(fmt.Sprintf("Error creating flag: %s. Press \"ctrl + c\" to quit.", e.Message))}
+			}
```

Comment:
maybe we clean this up when we do this for other steps also? https://app.shortcut.com/launchdarkly/story/239181/errors-during-setup-command-should-be-human-readable?vc_group_by=day&ct_workflow=all&cf_workflow=500010659

Reply:
Yeah, I think we could get something more generic when we do that one.